packages/icon-build-helpers/src/builders/react/builder.js

```diff
@@ -52,9 +54,13 @@ async function builder(meta, { cwd }) {
     },
   ];
 
-  const modules = meta.map(icon =>
-    createIconComponent(icon.moduleName, icon.descriptor)
-  );
+  const modules = meta.map(icon => {
+    let name = icon.moduleName;
+    if (isWin && name.match(/^\d/)) {
+      name = '_' + icon.moduleName;
```

Comment:
Is it the case that filenames can't start with numbers? 👀

Reply:
It's more of variable names can't start with numbers, yet how you build the names involves concatenating folder names, some of which start with numbers